overview/vaults.md

```diff
@@ -0,0 +1,41 @@
+{% hint style="warning" %} This documentation describes certain core aspects of MakerDAO's protocol that could change over time. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+# Maker Vaults
+
+Maker Vaults are a tool of the Maker Protocol through which users can create (mint) Dai against locked-up collateral.
+
+Users generate Dai by leveraging collateral and repay Dai to destroy their generated balance. This on-chain process ensures complete auditability of circulating Dai and its collateral support.
+
+Vaults are overcollateralized with a [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) maintained by owners to prevent collateral liquidation. This safeguards Dai's backing from volatile collateral devaluation.
+
+Additionally, a [debt ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) is imposed on each Vault type to prevent over-reliance on risky or illiquid collaterals.
+
+Vault owners who mint Dai incur a [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) on their balance. These fees manage the peg, fund Maker Protocol development, maintenance, and reward its governors.
+
+
+## RWA Vaults
+
+RWA (Real-World Asset) vaults enable users to collateralize real-world assets, like invoices and real estate, to generate Dai. Unlike traditional collateral limited to digital assets, RWA vaults broaden protocol's scope, enhancing diversification, stability, and financial inclusion.
```

Comment:
What invoices? Might be best to explain it as real-world assets which are first tokenized and then used to generate Dai. Also not sure what's meant by "financial inclusion"